Tangem/App/Services/UserWalletRepository/CommonUserWalletRepository.swift

```diff
@@ -241,6 +241,16 @@ class CommonUserWalletRepository: UserWalletRepository {
         save(cardViewModel.userWallet)
     }
 
+    func setSaving(_ enabled: Bool) {
+        if enabled {
+            if let selectedModel {
+                save(selectedModel)
+            }
+        } else {
+            clearNonSelectedUserWallets()
+        }
```

Comment:
```suggestion
       guard enabled else {
           clearNonSelectedUserWallets()
           return
       }
       
       if let selectedModel {
           save(selectedModel)
       }
```

в общем-то меньше и чище не прям сильно становится) но вложенность уменьшается. но в целом и с if-else норм